src/main/java/com/notably/logic/parser/DeleteCommandParser.java

```diff
@@ -54,6 +59,7 @@ public DeleteCommandParser(Model notablyModel, CorrectionEngine<AbsolutePath> pa
             throw new ParseException(String.format(ERROR_NO_MATCH_PATH, title));
         }
 
+        logger.info("DeleteCommand created");
```

Comment:
Perhaps you can be more explicit here. There are a lot of paths of execution in this `DeleteCommandParser#parse` execution. It'll be better if you can fine-tune the logger to log out the exact detail given the different paths that are taken!